### PR TITLE
AC 7934-166 TouchNet upay proxy unique paymentRequestNumber

### PR DESCRIPTION
**Pull request by franceslui**

### CODE REVIEW

File: src/Plugin/Commerce/PaymentGateway/OffsiteRedirect.php [hunk 1]
```diff
@@ -181,7 +182,9 @@ public function onNotify(Request $request):Response|null {
     }
 
     // 2. Check if the order can be loaded from the 'paymentRequestNumber'.
-    $order_id = $data['paymentRequestNumber'] ?? 0;
+    // Remove separator and suffix (if they exist) from paymentRequestNumber.
+    $order_id = explode(PaymentOffsiteForm::ORDER_ID_SUFFIX_SEPARATOR, $data['paymentRequestNumber'])[0]  ?? 0;
```
Comment (joelpittet) [inline src/Plugin/Commerce/PaymentGateway/OffsiteRedirect.php]: This looks good, could you remove that extra space before the `??`? And I know you can do array access off the `explode()` function but could you instead separate those two so it's easier to read?

**Review by joelpittet — CHANGES_REQUESTED** [review]
Super minor change request, you can do them and just push them to this branch to fix them.  Thanks @franceslui for making a PR

File: src/Plugin/Commerce/PaymentGateway/OffsiteRedirect.php (hunk 1 above)
Comment (franceslui) [reply src/Plugin/Commerce/PaymentGateway/OffsiteRedirect.php]: Thank you very much for your comment. I have followed your advice to make changes and pushed.

**Review by joelpittet — APPROVED** [review]
Thanks for the quick fix on that. Feel free to merge it into main and create a new `tag` for the changes

**Pull request merged**